composer.json

```
@@ -7,8 +7,9 @@
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
-    "wp-coding-standards/wpcs": "^2.3 | dev-develop#fca9d9e",
-    "automattic/vipwpcs": "^2.0.0"
+    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.0",
```

```suggestion
    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.x-dev",
```

Not blocker. Just name change suggestion.

---

@mukeshpanchal27 Can you explain the need for this change?